go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/grpc-ecosystem/grpc-gateway/v2
 
-go 1.21
+go 1.22.7
+
+toolchain go1.23.3
```

Comment:
Why does this library need to specify the toolchain and the go version so specifically?

Related: https://github.com/open-telemetry/opentelemetry-go/pull/5784

Reply:
This was changed on main again already